### ISSUE TITLE
Improve project templates?

### ISSUE THREAD

**Issue by tidyui** [PiranhaCMS/piranha.core#967]
Since it's almost 2020 I thought it would be a good time do improve and do some redesign of the project templates. Please leave feedback on:

* What you would like to be included in a good template
* What different types of templates you would like to see

**Comment by tidyui** [comment]
I'll go first 😀

First of I'd like to consolidate the templates into **one** web template available for both `Razor Pages` and `Mvc`, for example:

~~~ bash
dotnet new piranha.mvc
dotnet new piranha.razor
~~~

Instead of having different templates it would be nice to be order to seed different startup data, one for a blog setup, and one for a website for example.

**Comment by goranhell** [comment]
Sounds good. But also custom stuff. Custom pages, custom posts, etc...

**Comment by tidyui** [comment]
Yes, I’m currently looking at the most popular themes at theme forest to get a good idea of what functionality it should contain. I’m for example thinking about a dynamic approach with custom blocks for stuff like:

* Latest post (various layouts)
* Archive listing (various layouts)
* Related content (pages/posts)

**Comment by haidersafaa** [comment]
adding widget it will be very good it a region with content be show an all pages like footer costume menu (internal links(pages + post(s))+ external links) social media menu

**Comment by tidyui** [comment]
Hi @haidersafaa you can achieve this by adding a region to your site. See http://piranhacms.org/docs/content/sites for reference!

**Comment by haidersafaa** [comment]
thanks dear but can the user (not the developer  ) change the content of regions ?

**Comment by tidyui** [comment]
Yes it is changed from the manager just like page content when editing the site! Try a new project based on the blog template and you can find a region on the site where you can set logo, title and tag line (if I remember correctly)
